src/preprocessEnvVar.js

```diff
@@ -69,9 +69,11 @@ function preprocessEnvVar(config) {
     console.error(`ERROR (${name}): ${err.message}`);
     throw err;
   }
-  // eslint-disable-next-line no-console -- this is a CLI tool
-  console.info(`env var ${name}=${value} (${typeof value})`);
-  process.env[name] = value;
+  if (value) {
```

Comment:
```suggestion
  if (value != undefined) {
```

this checks that the value isn't `null` or `undefined` instead of falsy, meaning that empty strings, `0`, `false`, etc are accepted